openfisca_us/parameters/gov/states/in/tax/income/taxes/agi/rates.yaml

```diff
@@ -0,0 +1,11 @@
+description: IN adjusted gross income tax rate.
```

Comment:
Nit but I think we usually go with `rate` instead of `rates` when it's a single rate, could you rename?

Reply:
I saw that there were other "rates" where the rate only varied across years. I'm just focussing on 2021 right now but I expect Indiana's rate has changed in the past will change in the future. Ok for me to use `rates` now so we don't have to change the filename when that stuff is added?

Reply:
Also just saw your commit adding the previous year. Should I look back for the other `rates` I found and rename it to `rate` since it only varies across years?

Reply:
Will change.